nginx/templates/default.conf.template

```diff
@@ -1,19 +0,0 @@
-server {
```

Comment:
This file is still used in the Dockerfile (not Dockerfile.dev). So it should not be removed.

Reply:
Right, readded in b3a7afb0e476e70474146a71fff346ac22e7418b.